scripts/run-oneshot.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+accelerator=
+dryrun=
+filename=
+testtype=
+test_name=
+
+help()
+{
+  echo "Utility script to run one shots with XLML test."
+  echo "This script will:"
+  echo "- connect to the appropriate one shot cluster"
+  echo "- run the test"
+  echo
+  echo "Syntax: ./scripts/run-oneshot.sh [-a|d|f|h|t]"
+  echo "-a | --accelerator   The accelerator type, e.g. v2-8, v3-8, 8xv100, etc."
+  echo "-d | --dryrun        Dryrun. If set, then the test does not run and only prints commands."
+  echo "-f | --file          The filepath for the test."
+  echo "-h | --help          Print this help."
+  echo "-t | --type          Test type, e.g. [func|conv|functional|convergence]."
+}
+
+validate()
+{
+  if [ -z "$accelerator" ]
+  then
+    echo "Accelerator must be provided."
+    exit
+  fi
+  if [ -z "$filename" ]
+  then
+    echo "File name must be provided."
+    exit
+  fi
+  if [ -z "$testtype" ]
+  then
+    echo "Test type must be provided."
+    exit
+  fi
+
+  case $testtype in
+    "functional" | "func" )  testtype="func"
+                             ;;
+    "convergence" | "conv" ) testtype="conv"
+                             ;;
+    * )                      echo "Invalid test type provided."
```

Comment:
This isn't quite accurate: some of our tests have custom modes like keras API and perfzero.

Reply:
Good point, I removed that path so any test type can be added.